packages/headless/src/controllers/facets/category-facet/headless-category-facet.ts

```diff
@@ -138,7 +138,7 @@ export function buildCategoryFacet(
      */
     isSortedBy(criterion: CategoryFacetSortCriterion) {
       const request = getRequest();
-      return request.sortCriteria === criterion;
+      return request!.sortCriteria === criterion;
```

Comment:
If somehow a UI was initialized and no request was made before this function was called, would this throw an error?

If so, should we catch this error and throw one with a more descriptive message?

Reply:
Nope, it should always be a truthy value. The controller ensures that the request is registered,